utils/parseFlexibleBlocks.js

```diff
@@ -27,34 +27,37 @@ function stripCountry(html) {
 
 // For internal and external resources, use titleGeneral as SimpleCard title; otherwise, keep default title
 // For internal resource, check for leading `ftva/` string in uri and remove if it exists
+
 function parseSimpleCard(block) {
```

Comment:
@jendiamond It's not clear why you've removed much of the original parsing in this function; they were very specific. Your changes have caused regressions. Please revert or refactor.